docs/UserGuide.md

```diff
@@ -199,16 +206,19 @@ Shows a specific internship application entry and its details.
 
 Finds internship applications whose fields contain the corresponding given keywords (any match with any of the field and keyword pairs is displayed).
 
-**Format**: `find [n/SEARCH_STRING_COMPANY] [r/SEARCH_STRING_ROLE] [c/SEARCH_STRING_CYCLE] [s/SEARCH_STRING_STATUS] [i/SEARCH_STRING_NOTE]`
+**Format**: `find [n/SEARCH_STRING_COMPANY] [r/SEARCH_STRING_ROLE] [c/SEARCH_STRING_CYCLE] [s/SEARCH_STRING_STATUS] [d/SEARCH_STRING_DEADLINE][i/SEARCH_STRING_NOTE]`
 
 **Parameters**:
-* At least one keyword to find and field to be searched must be provided.
+* At least one of the optional fields must be provided.
 
 **Examples**:
-* `find n/Google` displays all applications with the company name containing Google as value
-* `find n/Apl` displays all applications with the company name containing Apl as a substring of value
-* `find c/Summer` displays all applications with the cycle containing Summer as value
-* `find n/Google c/Summer s/Pending` displays all applications with either the company name containing Google as value, OR the cycle containing Summer as value, OR the status with Pending as value
+* `find n/Google` displays all internship applications whose company name contains `Google`
+* `find n/Apl` displays all internship applications whose company name contains `Apl`
```

Comment:
Perhaps we can provide some examples here, such as `Apple, apple, Alphabet Limited` to show that it matches substrings.